spec/setup_test_database.rb

```diff
@@ -0,0 +1,5 @@
+require 'pg'
+p "Setting up test database..."
+
+connection = PG.connect(dbname: 'chitter_challenge_test')
+connection.exec("TRUNCATE chitter;")
```

Comment:
Could also use DROP TABLE here instead of TRUNCATE, which I think may be better if you are changing lots of columns in your table, as TRUNCATE just empties the data in the table as far as I'm aware.